### PR TITLE
Add parse response for neofs-cli container commands

### PR DESCRIPTION
**Pull request by yadro-vavdeev**
Add parse response for neofs-cli container commands

Signed-off-by: Vladimir Avdeev <v.avdeev@yadro.com>

### CODE REVIEW

**Comment by abereziny** [comment]
LGTM.

**Review by dansingjulia — DISMISSED** [review]

File: src/neofs_testlib/models/object.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class Object:
```
Comment (abereziny) [inline src/neofs_testlib/models/object.py:6]: python already have object class. 
Need to be renamed to StorageObject or something else

Comment (yadro-vavdeev) [reply src/neofs_testlib/models/object.py]: Done

**Comment by evgeniiz321** [comment]
Not relevant with the latest testcases version

**Pull request closed without merging**